src/ng2-modal.ts

```diff
@@ -1,7 +1,7 @@
 import {Component, Input, Output, EventEmitter, ElementRef, ViewChild} from "@angular/core";
 
 @Component({
-    selector: "modal",
+    selector: "ng2-modal",
```

Comment:
I prefer file to be named `ng2-modal`, but why do you think `<ng2-modal>` component name is a better name then `<modal>`? `<modal>` is simple and straightforward.